tests/integration/apiAttribution.test.js

```diff
@@ -0,0 +1,46 @@
+const {crawler, APICallCollector} = require('../../main.js');
+const assert = require('assert');
+
+async function main() {
+    const testOrigin = 'https://privacy-test-pages.glitch.me'; // set to https://first-party.example for local testing
+    let apiData;
+    try {
+        apiData = await crawler(new URL(`${testOrigin}/crawler/attribution/`), {
+            collectors: [new APICallCollector()],
+            // log: console.log,
+        });
+    } catch (e) {
+        assert(false, `Page load failed - ${e}`);
+    }
+
+    const expectedScripts = [
+        `${testOrigin}/crawler/attribution/`, // race condition
+        `${testOrigin}/crawler/attribution/entrypoints/createelement.js`,
+        `${testOrigin}/crawler/attribution/entrypoints/eval.js`,
+        // `${testOrigin}/crawler/attribution/entrypoints/dom0.js`, // https://app.asana.com/0/0/1204138450097419/f
```

Comment:
asana is not public, so I think we should either have those issues here on GH too, or add a short comment about the limitation